slang_frontend.cc

```diff
@@ -1039,6 +1070,12 @@ struct ProceduralVisitor : public ast::ASTVisitor<ProceduralVisitor, true, false
 		stmt.symbol.visit(*this);
 	}
 
+	void handle(const ast::BreakStatement &brk)
+	{
+		require(brk, !disableLoop_ins.empty())
```

Comment:
This should be a log_assert rather than require (require is for limitations of the frontend, log_assert for invariant violations)